onedal/basic_statistics/incremental_basic_statistics.py

```diff
@@ -155,6 +168,14 @@ def finalize_fit(self, queue=None):
         )
         options = self._get_result_options(self.options).split("|")
         for opt in options:
-            setattr(self, opt, from_table(getattr(result, opt)).ravel())
+            opt_value = self._input_xp.ravel(
```

Comment:
When running sklearnex example incremental_basic_statistics_dpctl.py leads to AttributeError: 'NoneType' object has no attribute 'ravel'